script.js

```diff
@@ -134,11 +134,30 @@ const urlconsctructor = function(){
         checkedline.push(i.value)
```

Comment:
This comments line `let seconds = currentTime - cache[key].time;` above:

I don't see `.time` set, then the difference is always NaN. Also, the difference is not in seconds, but milliseconds; the interval to check the expiration should be longer.

Reply:
`.coordinate` is a confusing name for variable to store time.

Also, this conversion `new Date(cache[key].coordinate)`, Date should be stored there anyway.

Reply:
if i understand correct, the JavaScript store it locally due to it in the loop. So after loop, it destroyed.

Reply:
I mean `cache[key].coordinate` - this is preserved as `cache` is a global variable.